controller.java

```diff
@@ -11,4 +11,8 @@ public static void main(String[] args) {
         // Print the result
         System.out.println("The sum of " + num1 + " and " + num2 + " is: " + sum);
     }
+
+    public static int add(int num1, int num2) {
+        return num1 + num2;
+    }
```

Comment:
!suggest change the function name to returnSum

Reply:
!suggest change the name to something else. Do not white space at the beginning.
```suggestion
    public static int sum(int num1, int num2) {
        return num1 + num2;
    }
```